File: accepted/437.cpp
#include <iostream>
#include <iomanip>
#include <string>
using namespace std;

int main() {
    int n_cases;
    cin >> n_cases;
    cin.ignore(); // Ignore the newline after the number

    for (int i = 0; i < n_cases; ++i) {
        string time_str;
        getline(cin, time_str);

        int h, m, s;
        sscanf(time_str.c_str(), "%d:%d:%d", &h, &m, &s);

        int total = h * 3600 + m * 60 + s;
        int start = 24 * 3600 - total;
        int sh = start / 3600;
        int sm = (start % 3600) / 60;
        int ss = start % 60;

        cout << setfill('0') << setw(2) << sh << ":"
             << setfill('0') << setw(2) << sm << ":"
             << setfill('0') << setw(2) << ss << endl;
    }
    return 0;
}

File: accepted/437.py
from sys import stdin

n_cases = int(stdin.readline()[:-1])


for _ in range(n_cases):
    h, m, s = map(int, stdin.readline().strip().split(":"))
    total = h * 3600 + m * 60 + s
    start = 24*3600 - total
    
    sh = start // 3600
    sm = (start % 3600) // 60
    ss = start % 60
    print("{:02d}:{:02d}:{:02d}".format(sh, sm, ss))
    